### PR TITLE
Puppeteer E2E test: Regenerate high-differenсe screenshots and decrease threshold

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: https://github.com/mrdoob/three.js/pull/24109

**Description**

Regenerate screenshots that produce at least 0.3% difference on at least one platform:
```
css3d_periodictable
misc_boxselection
misc_controls_pointerlock
misc_uv_tests
webgl2_multisampled_renderbuffers
webgl2_ubo
webgl_animation_skinning_blending
webgl_buffergeometry_compression
webgl_buffergeometry_drawrange
webgl_camera
webgl_camera_logarithmicdepthbuffer
webgl_clipping_advanced
webgl_custom_attributes_lines
webgl_custom_attributes_points2
webgl_custom_attributes_points3
webgl_framebuffer_texture
webgl_geometry_extrude_splines
webgl_gpgpu_birds_gltf
webgl_gpgpu_protoplanet
webgl_helpers
webgl_interactive_raycasting_points
webgl_lightningstrike
webgl_lights_pointlights
webgl_lights_spotlights
webgl_loader_collada_kinematics
webgl_loader_gcode
webgl_loader_gltf_transmission
webgl_loader_ldraw
webgl_loader_md2
webgl_loader_mmd_audio
webgl_loader_mmd_pose
webgl_loader_pcd
webgl_loader_pdb
webgl_loader_svg
webgl_loader_tilt
webgl_lod
webgl_marchingcubes
webgl_materials_blending
webgl_materials_blending_custom
webgl_materials_variations_lambert
webgl_materials_variations_phong
webgl_materials_variations_physical
webgl_materials_variations_standard
webgl_materials_variations_toon
webgl_materials_wireframe
webgl_multiple_canvases_circle
webgl_multiple_elements_text
webgl_nodes_loader_materialx
webgl_nodes_materials_instance_uniform
webgl_nodes_playground
webgl_points_dynamic
webgl_postprocessing_fxaa
webgl_postprocessing_pixel
webgl_postprocessing_ssr
webgl_postprocessing_advanced
webgl_raycaster_bvh
webgl_shaders_tonemapping
webgl_shadowmap_performance
webgpu_nodes_playground
webxr_vr_layers
webxr_vr_video
webxr_vr_sandbox
```
Generated using Chromium 109.0.5414.120 (revision 1070054, https://storage.googleapis.com/chromium-browser-snapshots/Win_x64/1070054/chrome-win.zip), stable channel on win64.

### CODE REVIEW

**Comment by LeviPesin** [comment]
Not sure what is the noise in webgl_camera...

File: test/e2e/puppeteer.js [hunk 1]
```diff
@@ -18,21 +18,36 @@ const exceptionList = [
...
 	'webaudio_visualizer', // audio can't be analyzed without proper audio hook
 
-	'webgl_effects_ascii', // blink renders text differently in every platform
-
 	'webxr_ar_lighting', // webxr
 
 	'webgl_worker_offscreencanvas', // in a worker, not robust
 
+	// Windows-Linux text rendering differences
+	// TODO: Fix these by setting a font in Puppeteer
```
Comment (LeviPesin) [inline test/e2e/puppeteer.js:30]: Will try to fix these in a separate PR.

File: test/e2e/puppeteer.js [hunk 2]
```diff
@@ -59,7 +76,7 @@ const chromiumChannel = 'stable'; // stable -> beta -> dev -> canary (Mac and Wi
 
 const port = 1234;
 const pixelThreshold = 0.1; // threshold error in one pixel
-const maxFailedPixels = 0.05; // at most 5% failed pixels
+const maxFailedPixels = 0.3; // at most 0.3% failed pixels
```
Comment (mrdoob) [inline test/e2e/puppeteer.js:79]: 0.3% or 3%?

Comment (LeviPesin) [reply test/e2e/puppeteer.js:79]: 0.3% -- more than 10x decrease.

**Comment by LeviPesin** [comment]
@mrdoob Can you please merge this PR (and https://github.com/mrdoob/three.js/pull/25386)? Both these PRs make the test more precise -- and also after their merge I will be able to make and test a PR fixing most of the examples in the exception list.

**Pull request merged**